saltstack/base/salt/prerequisites/jinja.sls

```diff
@@ -1,3 +1,9 @@
+{% if pillar['OS'] == 'redhat8' %}  
 install_jinja2:
   cmd.run:
```

Comment:
You can use this instead:

```
install_jinja2:
  pip.installed:
      - name: jinja2
```

pip3 is not always available or on the path - the above solution takes the pip available for Salt and uses that one - usually the one that is bound to the virtualenv Salt is running in. With that, you don't need the "if" to differentiate between OSes.

Reply:
Already done on another branch